script/check-config.sh

```diff
@@ -212,11 +213,11 @@ flags=(
 )
 check_flags "${flags[@]}"
 
-if [ "$kernelMajor" -lt 5 ] || [ "$kernelMajor" -eq 5 -a "$kernelMinor" -le 1 ]; then
+if [ "$kernelMajor" -lt 5 ] || [[ "$kernelMajor" -eq 5 && "$kernelMinor" -le 1 ]]; then
 	check_flags NF_NAT_IPV4
 fi
 
-if [ "$kernelMajor" -lt 5 ] || [ "$kernelMajor" -eq 5 -a "$kernelMinor" -le 2 ]; then
+if [ "$kernelMajor" -lt 5 ] || [[ "$kernelMajor" -eq 5 && "$kernelMinor" -le 2 ]]; then
```

Comment:
```suggestion
if [ "$kernelMajor" -lt 5 ] || { [ "$kernelMajor" -eq 5 ] && [ "$kernelMinor" -le 2 ]; }; then
```

(basically, these are probably only useful if you think it might be useful for this script to swap to `/bin/sh` someday, given what its purpose is)